pkg/api/types/build_types.go

```diff
@@ -0,0 +1,59 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package types
+
+type BuildCommandContext struct {
```

Comment:
Maybe we should keep the name consistent with other `Option` struct (i.e., `BuildCommandOptions`)? I know there might be something that are technically not `Option`, but it might confuse others that there is also a `BuildCommandOptions` somewhere.